lib/geometry.rb

```diff
@@ -1,18 +1,18 @@
 require "active_support/core_ext/module/delegation"
 require "memoist"
 
-require 'point'
-require 'segment'
-require 'vector'
-require 'polygon'
-require 'line'
-require 'bounding_box'
-require 'algorithms/point_in_polygon'
+require 'geometry/point'
+require 'geometry/segment'
+require 'geometry/vector'
+require 'geometry/polygon'
+require 'geometry/line'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.